frontend/src/concepts/k8s/pods/useFetchLogs.ts

```diff
@@ -14,7 +20,19 @@ const useFetchLogs = (podName: string, containerName: string, tail = 500): Fetch
     return getPodContainerLogText(namespace, podName, containerName, tail);
   }, [podName, containerName, namespace, tail]);
 
-  return useFetchState(callback, '');
+  const state = useFetchState(callback, '', { initialPromisePurity: true });
+  const [, , , refresh] = state;
+
+  React.useEffect(() => {
```

Comment:
I want to understand this, why aren't we using the `refreshRate` in `useFetchState`?
if we want to stop, shouldn't be enough to get that value into 0?

Reply:
This is possible -- I was co-locating code from the component as part of my cleanup and I did not consider it. Looking at `useFetchState` it looks possible even with the `initialPromisePurity` flag. I'll convert it over to clean up the code 👍